templates/addons/aurora/mysql-cf.yml

```diff
@@ -66,77 +58,81 @@ Resources:
+    DependsOn: ['{{logicalIDSafe .Name}}']
     Properties:
-      AliasName: !Sub 'alias/${AuroraDBCluster}'
+      AliasName: !Sub 'alias/${ {{- logicalIDSafe .Name -}} }'
       TargetKeyId: !Ref AuroraKMSCMK
-
   DBSubnetGroup:
     Type: 'AWS::RDS::DBSubnetGroup'
     Properties:
-      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster. 
+      DBSubnetGroupDescription: Group of Copilot private subnets for Aurora cluster.
       SubnetIds:
         !Split [',', { 'Fn::ImportValue': !Sub '${App}-${Env}-PrivateSubnets' }]
-
   DBClusterSecurityGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: The Security Group for the database cluster. 
+      GroupDescription: The Security Group for the database cluster.
       SecurityGroupIngress:
-        - ToPort: !Ref DBPort
-          FromPort: !Ref DBPort
+        - ToPort: 3306
+          FromPort: 3306
           IpProtocol: tcp
           Description: From Environment Security Group
-          SourceSecurityGroupId: 
-            Fn::ImportValue: 
+          SourceSecurityGroupId:
+            Fn::ImportValue:
               !Sub '${App}-${Env}-EnvironmentSecurityGroup'
-      VpcId: 
-        Fn::ImportValue: 
+      VpcId:
+        Fn::ImportValue:
           !Sub '${App}-${Env}-VpcId'
       SecurityGroupEgress:
         - IpProtocol: '-1'
           Description: Allow all outbound traffic by default
           CidrIp: 0.0.0.0/0
-
   AuroraSecret:
     Type: AWS::SecretsManager::Secret
     Properties:
-      Name: !Sub '/${App}/${Env}/${Name}/${DBTableName}'
-      Description: !Sub Aurora PostgreSQL main user secret for ${AWS::StackName}
+      Name: !Sub '/${App}/${Env}/${Name}'
+      Description: !Sub Aurora main user secret for ${AWS::StackName}
       GenerateSecretString:
-        SecretStringTemplate: '{"username": "postgres"}'
+        SecretStringTemplate: '{"username": "admin"}'
         GenerateStringKey: "password"
         ExcludePunctuation: true
         IncludeSpace: false
         PasswordLength: 16
-  
+  {{- if .ParameterGroup}}
+  # DBClusterParameterGroup:
+  #   Type: 'AWS::RDS::DBClusterParameterGroup'
+  #   Properties:
+  #     Description: !Ref 'AWS::StackName'
+  #     Family: 'aurora-mysql5.7'
+  #     Parameters:
+  #       character_set_client: 'utf8'
+  {{- else}}
   DBClusterParameterGroup:
     Type: 'AWS::RDS::DBClusterParameterGroup'
     Properties:
       Description: !Ref 'AWS::StackName'
-      Family: 'aurora-postgresql10'
+      Family: 'aurora-mysql5.7'
       Parameters:
-        client_encoding: 'UTF8'
-
-  AuroraDBCluster:
+        character_set_client: 'utf8'
+  {{- end}}
+  {{logicalIDSafe .Name}}:
     Type: 'AWS::RDS::DBCluster'
     Properties:
       MasterUsername:
-        !Join [ "", [ "{{resolve:secretsmanager:", !Ref AuroraSecret, ":SecretString:username}}" ]]
+        !Join [ "",  [ {{`'{{resolve:secretsmanager:'`}}, !Ref AuroraSecret, ":SecretString:username}}" ]]
```

Comment:
Ahhh, this is because the token for templates is set to `{{`. I think we can change this with the [`template.Delims`](https://golang.org/pkg/text/template/#Template.Delims) method; perhaps we can change them just for this template.

Reply:
Is there any reason why we would prefer `template.Delims` over just having ``{{ `{{content}}` }}``?